guide/interactions/select-menus.md

```diff
@@ -14,18 +14,18 @@ Select menus are part of the `MessageComponent` class, which can be sent via mes
 You can have a maximum of five `ActionRow`s per message, and one select menu within an `ActionRow`.
 :::
 
-To create a select menu, use the `MessageActionRow()` and `MessageSelectMenu()` builder functions and then pass the resulting object to `CommandInteraction#reply()` as `InteractionReplyOptions`:
+To create a select menu, use the `ActionRowBuilder()` and `SelectMenuBuilder()` functions and then pass the resulting object to `CommandInteraction#reply()` as `InteractionReplyOptions`:
 
 ```js {1,7-24,26}
-const { MessageActionRow, MessageSelectMenu } = require('discord.js');
+const { ActionRowBuilder, SelectMenuBuilder } = require('discord.js');
 
 client.on('interactionCreate', async interaction => {
-	if (!interaction.isCommand()) return;
+	if (!interaction.isChatInputCommand()) return;
```

Comment:
I find it odd that the above text states to use `CommandInteraction#reply()` yet we are only allowing chat input interactions through. This will work with context menus too.